packages/flutter_localizations/lib/src/l10n/material_de_CH.arb

```diff
@@ -18,7 +18,7 @@
   "selectedRowCountTitleOne": "1 Element ausgewählt",
   "selectedRowCountTitleOther": "$selectedRowCount Elemente ausgewählt",
   "cancelButtonLabel": "ABBRECHEN",
-  "closeButtonLabel": "SCHLIEẞEN",
+  "closeButtonLabel": "SCHLIESSEN",
```

Comment:
I have been trying to get the internal translations team to update this value (with little success). Your change may be reverted when the next translations update happens.

Reply:
Ack.